third-party/cbs/cbs.c

```diff
@@ -1,1050 +1,1050 @@
-/*
```

Comment:
If the file is not needed, delete it.

Reply:
Deleted

Reply:
@ReenigneArcher why was this file deleted? As of https://github.com/SunshineStream/Sunshine/blob/master/third-party/cbs/cbs.c the file is present inside the repository and it will be removed by this PR

Reply:
Because new ffmpeg library already has cbc.c implementation. I had to update ffmpeg and also added libmfx for qsv support